src/components/SavedSongs/SavedSong.js

```diff
@@ -0,0 +1,131 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import AudioButton from '../Buttons/AudioButton'
+import ButtonSecondary from '../Buttons/ButtonSecondary'
+import { ReactComponent as XIcon } from '../../assets/x.svg'
+import { ReactComponent as ResetIcon } from '../../assets/reset.svg'
+import { ReactComponent as BinIcon } from '../../assets/bin.svg'
+
+SavedSong.propTypes = {
+  stopPlayingSong: PropTypes.func,
+  artist: PropTypes.string.isRequired,
+  songTitle: PropTypes.string.isRequired,
+  id: PropTypes.string.isRequired,
+  deleteSavedSong: PropTypes.func,
+  toggleCurrentSongId: PropTypes.func,
+  isSongPlaying: PropTypes.bool.isRequired,
+  currentSongId: PropTypes.string,
+}
+
+export default function SavedSong({
+  stopPlayingSong,
+  artist,
+  songTitle,
+  id,
+  deleteSavedSong,
+  toggleCurrentSongId,
+  isSongPlaying,
+  currentSongId,
+}) {
+  const [toBeDeleted, setToBeDeleted] = useState(false)
+
+  return (
+    <>
+      {!toBeDeleted && (
+        <ListItem>
+          <ButtonWrapper>
+            <ButtonSecondary
+              id={id}
+              onClick={prepareDeletion}
+              data-testid="x-button"
```

Comment:
Maybe `prepare-delete-button` as a name?

Reply:
I agree this, or is this button used for something else?

Reply:
Good point! Thanks :)